src/Opulence/Framework/Databases/Console/Commands/FixMigrationsCommand.php

```diff
@@ -0,0 +1,176 @@
+<?php
+
+/*
+ * Opulence
+ *
+ * @link      https://www.opulencephp.com
+ * @copyright Copyright (C) 2017 David Young
+ * @license   https://github.com/opulencephp/Opulence/blob/master/LICENSE.md
+ */
+
+namespace Opulence\Framework\Databases\Console\Commands;
+
+use Exception;
+use Opulence\Console\Commands\Command;
+use Opulence\Console\Responses\IResponse;
+use Opulence\Databases\IConnection;
+use Opulence\QueryBuilders\MySql\QueryBuilder as MySqlQueryBuilder;
+use Opulence\QueryBuilders\PostgreSql\QueryBuilder as PostgreSqlQueryBuilder;
+use Opulence\QueryBuilders\QueryBuilder as BaseQueryBuilder;
+use RuntimeException;
+
+/**
+ * Defines the command that fixes migrations
+ */
+class FixMigrationsCommand extends Command
+{
+    /** @var string[] The list of migration classes */
+    private $allMigrationClasses = [];
+
+    /** @var string The name of the table to read and write to */
+    protected $tableName = '';
+
+    /** @var IConnection */
+    protected $connection;
+
+    /** @var BaseQueryBuilder */
+    protected $queryBuilder;
+
+    /**
+     * FixMigrationsCommand constructor.
+     *
+     * @param string           $tableName           The name of the table to read and write to
+     * @param string[]         $allMigrationClasses The list of migration classes
+     * @param IConnection      $connection
+     * @param BaseQueryBuilder $queryBuilder
+     */
+    public function __construct(
+        string $tableName,
+        array $allMigrationClasses,
+        IConnection $connection,
+        BaseQueryBuilder $queryBuilder
+    ) {
+        parent::__construct();
+
+        $this->tableName           = $tableName;
```

Comment:
Please fix the whitespace here and elsewhere so that there is only one space on either side of the equal sign